content/en/guides/integrations/xgboost.md

```diff
@@ -29,22 +29,22 @@ with wandb.init() as run:
   bst.fit(X_train, y_train, callbacks=[WandbCallback(log_model=True)])
 ```
 
-You can open [this notebook](https://wandb.me/xgboost) for a comprehensive look at logging with XGBoost and Weights & Biases
+You can open [this notebook](https://wandb.me/xgboost) for a comprehensive look at logging with XGBoost and W&B
 
 ## `WandbCallback` reference
 
 ### Functionality
 Passing `WandbCallback` to a XGBoost model will:
-- log the booster model configuration to Weights & Biases
-- log evaluation metrics collected by XGBoost, such as rmse, accuracy etc to Weights & Biases
+- log the booster model configuration to W&B
+- log evaluation metrics collected by XGBoost, such as rmse, accuracy etc to W&B
 - log training metrics collected by XGBoost (if you provide data to eval_set)
 - log the best score and the best iteration
-- save and upload your trained model to Weights & Biases Artifacts (when `log_model = True`)
+- save and upload your trained model to W&B Artifacts (when `log_model = True`)
 - log feature importance plot when `log_feature_importance=True` (default).
 - Capture the best eval metric in `wandb.Run.summary` when `define_metric=True` (default).
 
 ### Arguments
-- `log_model`: (boolean) if True save and upload the model to Weights & Biases Artifacts
+- `log_model`: (boolean) if True save and upload the model to W&B Artifacts
 
 - `log_feature_importance`: (boolean) if True log a feature importance bar plot
```

Comment:
These should start with capital letters and end in periods (through line 53). The pattern is that list items that are longer than a few words should be complete sentences.